provider/internal/builtin/provider.go

```diff
@@ -128,6 +128,8 @@ func (p *builtinProvider) Init(ctx context.Context, log logr.Logger, config prov
 		config:                             config,
 		tags:                               p.tags,
 		UnimplementedDependenciesComponent: provider.UnimplementedDependenciesComponent{},
+		locationCache:                      make(map[string]float64),
```

Comment:
Why float and not int?

Reply:
For some reason, in provider API, the value is float